<launch>
    <node pkg="pointcloud_utils" type="point_cloud_point_saver.py" name="pc_point_saver"
          required="true" output="screen" cwd="node">
          <param name="target_frame" value="base" />
          <param name="file_name"    value="innoviz_points" />
          <param name="folder_name"  value="points" />
          <param name="file_type" 	 value="xyz" />
		  <param name="cloud_topic" 		value="/invz_topic_1" />
    </node>
</launch> 